### PR TITLE
Ignore visitor messages after visitor has requested human

### PR DESCRIPTION
**Pull request by asdalol**
Check if visitor has reacted to feedback form and ignore all visitor messages after that.

### CODE REVIEW

File: .vscode/tags [hunk 1]
```diff
@@ -0,0 +1,124 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```
Comment (ktkiiski) [inline .vscode/tags]: ei tätä versionhallintaan?

File: server/bot.py [hunk 2]
```diff
@@ -171,11 +171,17 @@ def send_option_links(self, chat_id, message, help_text):
...
 
     def react_to_visitor_message(self, chat_id):
-        self.send_option_links(
-            chat_id,
-            "I apologize, I'm just a simple example bot uncapable of understanding human language! 😅",
-            "I only know what to do if you choose one of the options below!"
+        # Check if visitor has already answered to feedback
+        has_answered = self.api.search(
+            '/api/v5/users/{user_id}/chats/{chat_id}/messages'.format(chat_id=chat_id, **self.auth),
+            lambda message: message['response_value'] == 'positive_feedback' or message['response_value'] == 'negative_feedback'
```
Comment (ktkiiski) [inline server/bot.py]: `message['response_value'] in ('positive_feedback', 'negative_feedback')` ?

Comment (asdalol) [reply server/bot.py]: Nojoo totta

File: .vscode/tags (hunk 1 above)
Comment (asdalol) [reply .vscode/tags]: Jep, ei ollut muistanut pistää `.gitignore`:n mukaan tota

**Pull request merged**